### PR TITLE
docs: unified slack

### PR DESCRIPTION
**Pull request by alyssawada**
# What this PR does
Adds documentation to support the Unified IRM Slack integration

## Which issue(s) this PR closes

Related to [issue](https://github.com/grafana/technical-documentation/issues/961)

### CODE REVIEW

File: docs/sources/configure/integrations/references/slack/index.md
Comment (Konstantinov-Innokentii) [inline docs/sources/configure/integrations/references/slack/index.md:1]: Historically, we are keeping alert integration docs here. I mean it's things like AlertManager and Grafana Alerting.

File: docs/sources/configure/integrations/references/slack/index.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+There are two ways to upgrade to the new integration, depending on your current installation.
+
+### Upgrade from the OnCall Slack app to the IRM Slack app
+
+To upgrade from the OnCall Slack integration to the Grafana IRM Slack integration,
+you only need to approve the additional permissions required for Incident management functionality.
+Upgrading to the IRM Slack app will not affect your existing OnCall Slack-related configuration, including channels, users, and schedules.
+
+{{< admonition type="note" >}}
+The `/escalate` command is deprecated, please use `/grafana-irm escalate` instead. Refer to Available Slack commands for more information.
```
Comment (Konstantinov-Innokentii) [inline docs/sources/configure/integrations/references/slack/index.md:159]: Just a note - 99% we will change slash command before release

File: docs/sources/configure/integrations/references/slack/index.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+{{< admonition type="note" >}}
+If you already have the OnCall or Incident Slack integration installed, use the instructions in [Migrate to the IRM Slack integration](ref:slack-migration).
+{{< /admonition >}}
+
+Follow these steps to install the Slack integration from Grafana OnCall:
+
+1. Navigate to the **Settings** tab in Grafana OnCall.
+1. From the **ChatOps** tab, select **Slack** in the side menu.
+1. Click **Install Slack integration**.
```
Comment (Konstantinov-Innokentii) [inline docs/sources/configure/integrations/references/slack/index.md:124]: In latest UI this button is "Install integration"

File: docs/sources/configure/integrations/references/slack/index.md [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+
+{{< admonition type="note" >}}
+The `/escalate` command is deprecated, please use `/grafana-irm escalate` instead. Refer to Available Slack commands for more information.
+{{< /admonition >}}
+
+From OnCall, follow these steps to upgrade:
+
+1. Navigate to the **Settings** tab in Grafana OnCall.
+1. From the **ChatOps** tab, select **Slack** in the side menu.
+1. Click **Upgrade**
```
Comment (Konstantinov-Innokentii) [inline docs/sources/configure/integrations/references/slack/index.md:166]: We renamed this button to "Migrate"

File: docs/sources/configure/integrations/references/slack/index.md [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+
+{{< admonition type="note" >}}
+The `/escalate` and `/incident` Slack commands have been deprecated. Use `/grafana-irm` to learn more.
+{{< / admonition >}}
+
+The `/grafana-irm` Slack commands allow users and teams to respond to alert groups and collaborate on incidents directly from Slack.
+
+{{< admonition type="tip" >}}
+Use the 🤖 robot emoji Slack reaction to add important messages to the incident timeline.
+And use the 📝 memo emoji Slack reaction to add messages to the resolution notes.
```
Comment (Konstantinov-Innokentii) [inline docs/sources/configure/integrations/references/slack/index.md:294]: 📝 emoji works differently - it's added to message when user added it to resolution notes via message shortcut or modal. We are planning to rework this behavior to work like 🤖 emoji, but with no estimates.

File: docs/sources/configure/integrations/references/slack/index.md [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+| chat:write.customize |  Send messages as Grafana IRM with a customized username and avatar                                                | Allows @GrafanaIRM to post messages to Slack                                                                                                       |
+| chat:write.public    | Send messages to channels GrafanaI RM isn't a member of                                                            | Allows @Grafan IRM to post messages to Slack                                                                                                       |
+| reaction:read        | View emoji reactions and their associated content in channels and conversations that Grafana IRM has been added to | Allows @GrafanaIRM to monitor message events for the 🤖 emoji to be included in the incident timeline                                               |
+| reaction:write       | Add and edit emoji reactions                                                                                       | This permission is required for resolution notes. More specifically – to include messages with the 📝emoji on resolution notes                      |
```
Comment (Konstantinov-Innokentii) [inline docs/sources/configure/integrations/references/slack/index.md:378]: My bad here ' More specifically – to include messages with the 📝emoji on resolution notes' should be to "mark resolution notes messages"

File: docs/sources/configure/integrations/references/slack/index.md
Comment (alyssawada) [reply docs/sources/configure/integrations/references/slack/index.md:1]: Discussed in slack

**Review by Konstantinov-Innokentii — APPROVED** [review]

**Review by brojd — APPROVED** [review]

**Pull request merged**